src/wallet/wallet.h

```diff
@@ -136,9 +136,12 @@ enum WalletFlags : uint64_t {
 
     // will enforce the rule that the wallet can't contain any private keys (only watch-only/pubkeys)
     WALLET_FLAG_DISABLE_PRIVATE_KEYS = (1ULL << 32),
+
+    // Will enforce that the wallet should be blank
```

Comment:
In commit "[wallet] Support creating a blank wallet" (8061ea4611696b22d5d5dcdd73c1e053ffecc09d)

It would be nice to explain how the flag works. I'd suggest.

```c++
//! Flag set when a wallet contains no HD seed and no private keys, scripts,
//! addresses, and other watch only things, and is therefore "blank."
//!
//! The only function this flag has is to distinguish a blank wallet from
//! a newly created wallet when the wallet database is loaded, to avoid
//! initialization that should only happen on first run.
//!
//! This flag is also a mandatory flag to prevent previous versions of
//! bitcoin from opening the wallet, thinking it is newly created, and
//! improperly initializing it.
```